index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/styles.css" />
```

Comment:
Integrar el archivo `normalize.css`
```suggestion
    <link rel="stylesheet" href="./assets/styles.css" />
    <link rel="stylesheet" href="./assets/normalize.css" />
```